### PR TITLE
Dex 692 admin initiated collabs say so

### PR DESCRIPTION
**Pull request by Atticus29**

### CODE REVIEW

File: src/components/AuthenticatedAppHeader/NotificationsPane.jsx [hunk 1]
```diff
@@ -73,9 +75,27 @@ export default function NotificationsPane({
...
                   <div style={{ display: 'flex' }}>
                     <Avatar src={shane} variant="circular" />
-                    <Text style={{ maxWidth: 200, margin: '0 20px' }}>
-                      {`${senderName} sent you a collaboration request`}
-                    </Text>
+                    {senderName !== 'N/A' && (
+                      <Text
+                        style={{ maxWidth: 200, margin: '0 20px' }}
+                      >
+                        {`${senderName}`}
```
Comment (brmscheiner) [inline src/components/AuthenticatedAppHeader/NotificationsPane.jsx:82]: is there a space between senderName and "sent you a collaboration request"? Just making sure. There is also a way to inject sendername directly into the intl message as a value.

File: src/pages/notifications/Notifications.jsx [hunk 2]
```diff
@@ -70,6 +72,15 @@ export default function Notifications() {
             <List>
               {safeNotifications.map(notification => {
                 const read = get(notification, 'is_read', false);
+                const hasSenderName = Boolean(
```
Comment (brmscheiner) [inline src/pages/notifications/Notifications.jsx:75]: since you use `&&` this is already a boolean

**Review by brmscheiner — APPROVED** [review]
small things feel free to merge after addressing!

File: src/components/AuthenticatedAppHeader/NotificationsPane.jsx (hunk 1 above)
Comment (Atticus29) [reply src/components/AuthenticatedAppHeader/NotificationsPane.jsx:82]: Added a space. Done.

File: src/pages/notifications/Notifications.jsx (hunk 2 above)
Comment (Atticus29) [reply src/pages/notifications/Notifications.jsx]: Done

**Pull request merged**